sdk/tests/conformance/rendering/blending.html

```diff
@@ -146,7 +146,8 @@
         debug('Blending for RGBA8:');
 
         const gl = CreateContext();
-        fb = CreateValidFb(gl, [[gl.RGBA8, gl.RGBA, gl.UNSIGNED_BYTE]]);
+        // Do not use RGBA8 enum on WebGL 1.0
+        fb = CreateValidFb(gl, [[gl.texImage3D ? gl.RGBA8 : undefined, gl.RGBA, gl.UNSIGNED_BYTE]]);
```

Comment:
```suggestion
        fb = CreateValidFb(gl, [[gl.texImage3D ? gl.RGBA8 : gl.RGBA, gl.RGBA, gl.UNSIGNED_BYTE]]);
```

Reply:
`undefined` works but I think it's nice to just ask for the right thing up-front.

Reply:
Should similar updates be done to `RGBA16F` and `RGBA32F` below? Those rely on `undefined` being a special case in `CreateValidFb`.